File: 02-Patterns/pattern10/solution.js
/*

*
**
***
****
*****
****
***
**
*


*/

let n = 9; //number of rows
let breakpoint = 5; //row where the number of stars started decreasing

for (let i = 1; i <= 9; i++) {
  let stars = i;
  if (stars > breakpoint) stars = 2 * breakpoint - i;
  for (j = 0; j < stars; j++) {
    process.stdout.write("* ");
  }
  console.log("");
}

File: 02-Patterns/pattern10/solution.py
"""

*
* *
* * *
* * * *
* * * * *
* * * *
* * * 
* *
*

"""

n = 9; #number of rows
breakpoint = 5; #row where the number of stars started decreasing

for i in range(1, 10):
    stars = i
    if stars > breakpoint: stars = 2 * breakpoint - i
    for i in range(stars):
        print("*", end=" ")
    print("")